phoenicis-scripts/src/main/java/org/phoenicis/scripts/engine/implementation/PolyglotScriptEngine.java

```diff
@@ -43,7 +43,8 @@ public PolyglotScriptEngine(String language, Map<String, String> options) {
         this.language = language;
         this.context = Context.newBuilder(language)
                 .allowExperimentalOptions(true)
-                .options(options).allowHostAccess(true).build();
+                .allowHostAccess(true)
```

Comment:
I tried to replace by `allowHostAccess(true)` by something else because it seems to be deprecated but I couldn't find a working replacement operation.